skypalwarmup/JSUserRegistrationService/src/main/java/application/dto/UserInputDTO.java

```diff
@@ -0,0 +1,28 @@
+package application.dto;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Copyright (c) 2016-2018, Jumia.
+ */
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserInputDTO {
```

Comment:
I was thinking about if we needed to return a UserDTO upon successful registration, the fields would be different in case we wanted to include the user id or other fields we generate and remove the password field.

Reply:
No problem, but In this case it could be simply a "User" entity with the ID and without the password or a masked one :)